Renumber transport parameters (#1946)
quicwg/base-drafts: draft-ietf-quic-transport.md

  Values below 1200 are invalid.  This limit only applies to protected packets
  ({{packet-protected}}).

initial_max_data (0x0004):

: The initial maximum data parameter contains the initial value for the maximum
  amount of data that can be sent on the connection.  This parameter is encoded
  as an unsigned 32-bit integer in units of bytes.  This is equivalent to
  sending a MAX_DATA ({{frame-max-data}}) for the connection immediately after
  completing the handshake. If the transport parameter is absent, the connection
  starts with a flow control limit of 0.

initial_max_stream_data_bidi_local (0x0005):

> : The initial stream maximum data for bidirectional, locally-initiated streams

Reviewer: ```suggestion
: The parameter specifying the initial limit on stream data for bidirectional, locally-initiated
  streams is a 32-bit integer that contains the initial flow control limit for
```

Reviewer: This parameter is a 32-bit integer specifying the initial flow control limit for locally-initiated bidirectional streams. This limit applies to newly created bidirectional streams opened by the endpoint that sends the transport parameter.

  parameter is equivalent to sending a MAX_STREAMS ({{frame-max-streams}}) of
  the corresponding type with the same value.

ack_delay_exponent (0x000a):

: The ACK delay exponent is an 8-bit unsigned integer value indicating an

>   exponent used to decode the ACK Delay field in the ACK frame, see

Reviewer: ```suggestion
  exponent used to decode the ACK Delay field in the ACK frame ({{frame-ack}}).
```

>   {{frame-ack}}.  If this value is absent, a default value of 3 is assumed

Reviewer: ```suggestion
  If this value is absent, a default value of 3 is assumed
```

  (indicating a multiplier of 8).  The default value is also used for ACK frames
  that are sent in Initial and Handshake packets.  Values above 20 are invalid.

max_ack_delay (0x000b):

> : The maximum ACK delay is an 8 bit unsigned integer value indicating the

Reviewer: ```suggestion
: The maximum ACK delay is an 8-bit unsigned integer value indicating the
```

  maximum amount of time in milliseconds by which the endpoint will delay
  sending acknowledgments.  If this value is absent, a default of 25
  milliseconds is assumed.

disable_migration (0x000c):

> : The disable migration transport is included if the endpoint does not support

Reviewer: ```suggestion
: The disable migration transport parameter is included if the endpoint does not support
```
